### PR TITLE
Raise error when invalid parameters are requested

### LINKED ISSUE #8: Handling invalid parameters
When a parameter is requested through `get_parameters` that does not exist, we are currently ignoring the `InvalidParameters` key in the boto3 response which would tell us of our mistake.

For instance:

```
parameter_store = ParameterStore(ssm_client)
parameters = parameter_store.get_parameters(["/no/nope/noooo"])
assert len(parameters) == 0
```

Should we raise an error in this case? AWS tells us in the response, with the `InvalidParameters` key, that this request cannot be fulfilled so that seems helpful to me.

Regarding `get_parameters_by_path`, how do people check if required parameters are present? Should we maybe add an optional parameter to check for required parameters?

### PR DESCRIPTION
**Pull request by mpaddle**
Fixes #8 

When this is merged, let's push 1.0.0 🎉 :shipit:

### CODE REVIEW

File: python_aws_ssm/parameters.py [hunk 1]
```diff
@@ -31,11 +41,20 @@ def get_parameters(self, ssm_key_names: List[str]) -> Dict[str, Optional[str]]:
...
         """
 
-        retrieved_parameters = self.client.get_parameters(
+        response: Dict = self.client.get_parameters(
             Names=ssm_key_names, WithDecryption=True
-        ).get("Parameters")
+        )
+        if (
+            response.get("InvalidParameters")
+            and type(response["InvalidParameters"]) is list
```
Comment (vhemt) [inline python_aws_ssm/parameters.py]: this seems a bit confusing, but I didn't check the client response. Is there a case where we have `InvalidParameters` with a different type than a list?

Comment (mpaddle) [reply python_aws_ssm/parameters.py]: That's a fair question. I haven't seen a case like that so maybe this is overly defensive. Should I change it to `if response.get("InvalidParameters"):`?

Comment (vhemt) [reply python_aws_ssm/parameters.py]: it's probably better 👍

Comment (mpaddle) [reply python_aws_ssm/parameters.py]: Fixed 👍

**Pull request merged**